llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -540,6 +540,48 @@ void StoreInst::dump() const {
   dump(dbgs());
   dbgs() << "\n";
 }
+#endif // NDEBUG
+
+Value *ReturnInst::create(Value *RetVal, Instruction *InsertBefore,
```

Comment:
Return type should be ReturnInst * to be consistent with definition and other create*Inst()

Reply:
Ah yes, in this case we can return `RetunInst *`. In some other cases it gets folded by the IRBuilder, but not here.